etc/workflows/fast.xml

```diff
@@ -311,7 +220,7 @@
       exception-handler-workflow="partial-error"
       description="Archiving">
       <configurations>
-        <configuration key="source-tags">archive</configuration>
+        <configuration key="source-flavors">*/source,dublincore/*,security/*</configuration>
       </configurations>
```

Comment:
Are none of the generated encodes or images archived? Or, are working files snapshot'd along the way except for the original input files? Does this final snpashot override the previous temporary working snapshots that contained encodes and images?

Reply:
Encoded material is not and was never being archived by this worklfow. The archive tag was previously also added only to these specific files and the only reason it was used at all was that other workflows were expecting it to exist despite being optional. That problem has been fixed in the meantime since now every element retrieved from the asset manager automatically gets the `archive` tag. Hence, it was no longer necessary to use it here which means one less operation to run.